src/Metadata/Parameter.php

```diff
@@ -89,6 +92,14 @@ public function getPriority(): ?int
         return $this->priority;
     }
 
+    /**
+     * @return Constraint|Constraint[]|null
+     */
+    public function getConstraint(): array|Constraint|null
```

Comment:
I suggest to always manipulate an array of constraints

Reply:
For DX it's quite nice no? 

```
#[Parameter(constraint: new NotBlank())]
```

Reply:
it should rather be `new All(new NotBlank(), new Email())` so what about removing the `array`?

Reply:
actually `All` can't be used like this, we need to keep the array.

Reply:
So `constraints`?
```
#[Parameter(constraints: [new NotBlank(), new Email()])
```